flink-staging/flink-gelly/src/main/java/org/apache/flink/graph/GraphCsvReader.java

```diff
@@ -0,0 +1,502 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.graph;
+import com.google.common.base.Preconditions;
+import org.apache.flink.api.common.functions.MapFunction;
+import org.apache.flink.api.java.DataSet;
+import org.apache.flink.api.java.io.CsvReader;
+import org.apache.flink.api.java.tuple.Tuple2;
+import org.apache.flink.api.java.tuple.Tuple3;
+import org.apache.flink.core.fs.Path;
+import org.apache.flink.api.java.ExecutionEnvironment;
+import org.apache.flink.types.NullValue;
+/**
+ * A class to build a Graph using path(s) provided to CSV file(s) with edge (vertices) data
+ * The class also configures the CSV readers used to read edges(vertices) data such as the field types,
+ * the delimiters (row and field),  the fields that should be included or skipped, and other flags
+ * such as whether to skip the initial line as the header.
+ * The configuration is done using the functions provided in The {@link org.apache.flink.api.java.io.CsvReader} class.
+ */
+public class GraphCsvReader<K,VV,EV>{
+
+	private final Path path1,path2;
+	private final ExecutionEnvironment executionContext;
+	protected CsvReader EdgeReader;
+	protected CsvReader VertexReader;
+	protected MapFunction<K, VV> mapper;
+
+//--------------------------------------------------------------------------------------------------------------------
+
+	public GraphCsvReader(Path path1,Path path2, ExecutionEnvironment context)
+	{
```

Comment:
Also, @vasia was talking about some general coding style rules in one of the previous comments... The way we add the opening block brackets must be consistent. So here, after public GraphCsvReader(...) { //open the bracket on the same line.

Please look in the rest of the document for similar issues...